<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchbutton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchedit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="searchresult">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>850</width>
      <height>800</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:24pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Source Code Pro','monospace'; color:#000000;&quot;&gt;Номер в базе данных:&lt;br /&gt;Название:&lt;br /&gt;Жанр:&lt;br /&gt;Тип кино:&lt;br /&gt;Дата премьеры:&lt;br /&gt;Продолжительность в минутах:&lt;br /&gt;Описание:&lt;br /&gt;Постер:&lt;br /&gt;Изображение:&lt;br /&gt;Трейлер:&lt;br /&gt;Режиссер:&lt;br /&gt;Главные актеры:&lt;br /&gt;Все актеры:&lt;br /&gt;Количество сезонов / серий:&lt;br /&gt;Рейтинг на IMDb:&lt;br /&gt;Рейтинг на Rotten Tomatoes:&lt;br /&gt;Рейтинг на КиноПоиск:&lt;br /&gt;Где смотреть:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="film_poster">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="film_image">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>500</y>
      <width>360</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="descriptionbutton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trailerbutton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Трейлер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="castbutton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Актеры</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="adding">
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="add_info_action"/>
   </widget>
   <widget class="QMenu" name="helping">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="help_action"/>
   </widget>
   <widget class="QMenu" name="about_program">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="about_program_action"/>
   </widget>
   <widget class="QMenu" name="changing">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="change_info_action"/>
   </widget>
   <widget class="QMenu" name="filtering">
    <property name="title">
     <string>Фильтр</string>
    </property>
    <addaction name="filter_info_action"/>
   </widget>
   <addaction name="filtering"/>
   <addaction name="changing"/>
   <addaction name="adding"/>
   <addaction name="helping"/>
   <addaction name="about_program"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="change_info_action">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-edit-16.png</normaloff>icons/icons8-edit-16.png</iconset>
   </property>
   <property name="text">
    <string>Править информацию</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="add_info_action">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-add-new-16.png</normaloff>icons/icons8-add-new-16.png</iconset>
   </property>
   <property name="text">
    <string>Добавить информацию</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="filter_info_action">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-filter-16.png</normaloff>icons/icons8-filter-16.png</iconset>
   </property>
   <property name="text">
    <string>Фильтровать информацию</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="about_program_action">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-information-16.png</normaloff>icons/icons8-information-16.png</iconset>
   </property>
   <property name="text">
    <string>Информация</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
  <action name="help_action">
   <property name="icon">
    <iconset>
     <normaloff>icons/icons8-help-16.png</normaloff>icons/icons8-help-16.png</iconset>
   </property>
   <property name="text">
    <string>Помощь</string>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
